docs/tutorials/intro.md

```diff
@@ -70,102 +72,72 @@ algokit init
 
-Once the playground project is created, you will notice in the `hello_world` folder a file named `demo.py` which is a simple example of using AlgoKit to deploy and make a call to the `helloworld.py` smart contract on the LocalNet instance started earlier.
+Since you have just instantiated an official AlgoKit template, each official smart contract template includes an interactive codespace walkthrough powered by [CodeTours](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour).
 
-![AlgoKit Playground Demo](../imgs/algokitdemo.png)
+To start the interactive walkthrough, install the extension and click on the green play button in the left bottom corner of the `Explorer` pane as demonstrated on the screenshot below (See `Getting Started with Your AlgoKit Project`).
 
-Right clicking on this file and selecting `Run Python File in Terminal` will deploy the `HelloWorldApp` smart contract and then call it passing the parameter `name` with a value of `Beaker`. Alternatively, you can hit F5 (or whatever keyboard shortcut is set in VS Code for running the debugger) while you are viewing the `helloworld.py` file and it will:
+![CodeTour Play Button](../imgs/algokitcodetour.png)
 
-1. Start LocalNet
-2. Build the smart contract
-3. Deploy and call the smart contract (`demo.py`)
+Upon execution, the walkthrough will guide you through the key components of your fresh AlgoKit project instance, which is an important prerequisite to learn before running and deploying your first smart contract. Certainly, not because its hard to learn, but because it will save you a lot of time and efforts if you set the right foundation first.
 
-This should produce something similiar to the following in the VSCode terminal.
+## Run the Demo Application 🏃‍♂️
 
-```shell
-(playground-py3.11) User@Algo-User-MBP myalgokit project % "/Users/user/code/algokit/myalgokit pro
-ject/.venv/bin/python" "/Users/user/code/algokit/myalgokit project/playground/hello_world/demo.py"
-Deployed app in txid SBNIJYZUOXVPXDFJHAVDDCO5TQ4WZ7P37QOOQM6CSVSMIURETHNQ
-        App ID: 11
-        Address: 377KDIVHB7K2LFFJIPUPQFWJGVD36MAP4EI6Y2HAGRAW6JPK4MDUBC3YVY
+Once the `Algorand Python` based project is created, you can get started by building and deploying your first smart contract. There are a few possible ways to do so as demonstrated on the following video.
 
-Hello, Beaker
-```
+[![Watch the video](https://ipfs.algonode.xyz/ipfs/Qmc9mRaPoDyhUFmek4ETxVfKUKzUg9pf3Tss5xwkBGdQis)](https://ipfs.algonode.xyz/ipfs/Qma6gNqxsSFc9Jbh8kBTZyVLv5gqFj1xnrsjoeT6MAAwCw/)
 
-The App ID of of the deployed contract and its Algorand address is displayed, followed by the message returned from the smart contract call (`Hello, Beaker`).
+As shown above, click on clicking on `Run and Debug` and then `Build & Deploy contracts`. Alternatively, you can hit F5 (or whatever keyboard shortcut is set in VS Code for running the debugger). While you are viewing the `contract.py` file - it will:
```

Comment:
```suggestion
As shown above, click on `Run and Debug` and then `Build & Deploy contracts`. Alternatively, you can hit F5 (or whatever keyboard shortcut is set in VS Code for running the debugger). While you are viewing the `contract.py` file - it will:
```